adoptium-updater-parent/adoptium-api-v3-updater/pom.xml

```diff
@@ -10,6 +10,7 @@
     </parent>
 
     <artifactId>adoptium-api-v3-updater</artifactId>
+    <packaging>pom</packaging>
```

Comment:
this is not pom packaging, probably jar

Reply:
Interestingly I get:

`The packaging plugin for this project did not assign a main file to the project but it has attachments. Change packaging to 'pom'. -> [Help 1]`

I'll do some digging

Reply:
OK, downgrading to 2.5.2 for the install plugin gets past this so I'll do that as a starting point, will need to revisit this later.